config/config.go

```diff
@@ -67,6 +67,9 @@ type ServerConfig struct {
 	DbExtraNames []string // Additional databases that should be fetched (determined by additional databases in db_name)
 	DbAllNames   bool     // All databases except template databases should be fetched (determined by * in the db_name list)
 
+	// Postgres data directory, as used for system stats (autodetected if unset)
+	DataDirectory string `ini:"data_directory"`
```

Comment:
I wonder if we should use the `db_` prefix? (and also call the env variable `DB_DATA_DIRECTORY`?)

That makes it clear its not the data for the collector, but rather the data directory of the DB we're monitoring.